site/src/testHelpers/entities.ts

```diff
@@ -2777,29 +2777,24 @@ export const MockHealth = {
       "netcheck: [v1] measureAllICMPLatency: listen ip4:icmp 0.0.0.0: socket: operation not permitted",
       "netcheck: [v1] report: udp=true v6=false v6os=true mapvarydest=false hair= portmap= v4a=34.71.26.24:55368 derp=999 derpdist=999v4:2ms,10007v4:175ms,10008v4:112ms,10009v4:139ms",
     ],
-    error: null,
   },
   access_url: {
     access_url: "https://dev.coder.com",
     healthy: true,
     reachable: true,
     status_code: 200,
     healthz_response: "OK",
-    error: null,
   },
   websocket: {
     healthy: true,
-    response: {
-      body: "",
-      code: 101,
-    },
-    error: null,
+    body: "",
+    code: 101,
   },
   database: {
     healthy: true,
     reachable: true,
-    latency: 92570,
-    error: null,
+    latency: "92570",
```

Comment:
Can we get rid of this field?

Reply:
Many places in this API response are discussable. I second the question.